transformer/diff.go

```diff
@@ -2,76 +2,112 @@ package transformer
 
 import (
 	"fmt"
+	"regexp"
+	"strconv"
```

Comment:
Refactor Suggestion: Consider extracting the string to a constant to improve code readability.